ansible_collections/arista/avd/roles/eos_designs/templates/core_interfaces/ethernet-interfaces.j2

```diff
@@ -32,7 +32,18 @@ ethernet_interfaces:
 {%         endif %}
 {%         if switch.underlay_routing_protocol in ["ospf", "ospf-ldp"] %}
     ospf_network_point_to_point: true
-    ospf_area: {{ underlay_ospf_area }}
+{%             if p2p_interface.ospf_area is arista.avd.defined  %}
+    ospf_area: {{  p2p_interface.ospf_area }}
+{%             endif %}
+{%             if p2p_interface.ospf_authentication is arista.avd.defined %}
+    ospf_authentication: {{ p2p_interface.ospf_authentication | arista.avd.default(ospf_authentication) }}
```

Comment:
Here this line will never be read if only global `ospf_authentication` exists because of the `if` statement. is it what is expected?